allure-rest-assured/src/main/java/io/qameta/allure/restassured/AllureRestAssured.java

```diff
@@ -77,7 +89,7 @@ public Response filter(final FilterableRequestSpecification requestSpec,
         final Prettifier prettifier = new Prettifier();
 
 
-        final HttpRequestAttachment.Builder requestAttachmentBuilder = create("Request", requestSpec.getURI())
+        final HttpRequestAttachment.Builder requestAttachmentBuilder = create(requestAttachmentName, requestSpec.getURI())
```

Comment:
I believe checkstyle fails on this and 109 line. You may want to run a corresponding gradle task and fix all the issues.